assets/airports/mdsd.json

```diff
@@ -0,0 +1,5080 @@
+                "KODIX": ["KODIX"]
+            },
+            "draw": [
+                ["CDO", "PITOX", "KODIX"]
+            ]
+        },
+        "KODIX2": {
+            "icao": "KODIX2",
+            "name": "Kohdeex Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "CDO", "PITOX"],
+            "exitPoints": {
+                "KODIX": ["KODIX"]
+            },
+            "draw": []
+        },
+        "LECKY": {
+            "icao": "LECKY",
+            "name": "Lecky",
+            "suffix": {"17":"1J", "35":"1L"},
+            "rwy": {
+                "17": ["SD130", "SD131"],
+                "35": ["^SD140", "SD142", "SD143"]
+            },
+            "body": [],
+            "exitPoints": {
+                "LECKY": ["LECKY"]
+            },
+            "draw": [
+                ["SD131", "LECKY"],
+                ["SD143", "LECKY"]
+            ]
+        },
+        "LECKY1": {
+            "icao": "LECKY1",
+            "name": "Lecky One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "CDO"],
+            "exitPoints": {
+                "LECKY": ["LECKY"]
+            },
+            "draw": [
+                ["CDO", "LECKY"]
+            ]
+        },
+        "LECKY2": {
+            "icao": "LECKY2",
+            "name": "Lecky Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "_CDO237015"],
+            "exitPoints": {
+                "LECKY": ["LECKY"]
+            },
+            "draw": []
+        },
+        "MIROX": {
+            "icao": "MIROX",
+            "name": "Mirox",
+            "suffix": {"17":"1J", "35":"1L"},
+            "rwy": {
+                "17": ["SD130", "SD131", ["SD132", "A40-"]],
+                "35": ["^SD140", "SD144", "SD145", ["SD146", "A30"], ["SD148", "A60+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "MIROX": ["MIROX"]
+            },
+            "draw": [
+                ["SD132", "MIROX"],
+                ["SD146", "SD148", "MIROX"]
+            ]
+        },
+        "PIVAR1": {
+            "icao": "PIVAR1",
+            "name": "Peevar One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "CDO"],
+            "exitPoints": {
+                "PIVAR": ["PIVAR"]
+            },
+            "draw": [
+                ["CDO", "PIVAR"]
+            ]
+        },
+        "PIVAR2": {
+            "icao": "PIVAR2",
+            "name": "Peevar Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "_CDO099015"],
+            "exitPoints": {
+                "PIVAR": ["PIVAR"]
+            },
+            "draw": []
+        },
+        "POBEX1": {
+            "icao": "POBEX1",
+            "name": "Pohbex One",
+ 
```

Comment:
So here's an interesting case where on the `17` arrivals, they cross `BIBOK` at 7000 or better, versus 9000 or better on the `35` arrivals. Since the _restrictions_ of the entry point is dependent on the _runway assignment_, we actually will have no choice but to list the entire thing out in the runway sections, leaving the body as well as each entry point with no waypoints.